src/azul/chalice.py

```diff
@@ -64,6 +64,8 @@ class GoneError(ChaliceViewError):
 
 
 class AzulChaliceApp(Chalice):
+    # https://github.com/DataBiosphere/azul-chalice/blob/8d296f4cb1a7268300818cf53fc63d9cf310d471/chalice/constants.py#L57
```

Comment:
Is this comment acceptable, what enhancements can be made to it to improve it?

Reply:
Why so complicated?